packages/browser/__tests__/ClientAuthentication.spec.ts

```diff
@@ -318,4 +402,243 @@ describe("ClientAuthentication", () => {
       }
     });
   });
+
+  describe("getCurrentIssuer", () => {
+    it("returns null no current session is in storage", async () => {
+      const clientAuthn = getClientAuthentication({});
+
+      await expect(clientAuthn.getCurrentIssuer()).resolves.toBeNull();
+    });
+
+    // ts-ignore is used to mock out local storage.
+    /* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
This appears to be a couple of lines too high? (Which I would assume would result in it not working?)

Also, I'd generally prefer type assertions over ts-ignore, to make sure we only catch expected errors. (So in these cases, something like `(new LocalStorageMock(/* ... */)) as typeof window.localStorage` or something.)

Reply:
Ah wait, I suppose `window.localStorage` is readonly? In that case, what you could do is add a type assertion to `window` I think, e.g. `(window as any).localStorage`.

Reply:
Yeah exactly it's because `localStorage` is readonly. I'll change it to your suggestion, which is also how Pat approached it in another file.

Reply:
9489169716956c6376af4bf92fdd2cb3abafc4df